src/Query.php

```diff
@@ -223,18 +230,7 @@ function ($item) use ($instance) {
 	 */
 	public function __call($method, $parameters)
```

Comment:
Is there any point in the `__call` method after this change?

Reply:
at this point no. But I would like to keep it, because something custom may be needed here.

Reply:
I think we do want `$passThrough` etc. but for now maybe it's better to simplify and remove it for now - we can work on a approach to exposing the databaseQuery methods later on.

Reply:
Actually there is another reason for it: chaining method calls, e.g.: `$this->from($this->model->getTable())->select($columns)->order('id DESC')->setLimit(1);
`